consultation_analyser/consultations/migrations/0025_generate_slugs_for_existing.py

```diff
@@ -0,0 +1,31 @@
+# Populate slug field for existing ProcessingRuns
+
+import faker as _faker
+from django.db import migrations
+
+faker = _faker.Faker()
+
+
+def populate_processing_run_slug(apps, schema_editor):
+    ProcessingRun = apps.get_model("consultations", "ProcessingRun")
+    for run in ProcessingRun.objects.all():
```

Comment:
Why not call run.save()? this code will run in the model, won't it?

Reply:
No - turns out you can't call custom `save()` in migrations: https://docs.djangoproject.com/en/5.1/topics/migrations/#historical-models (which I learnt the hard way yesterday)

Reply:
oh how annoying!